apps/cow-amm-deployer/src/components/DepositForm.tsx

```diff
@@ -153,10 +147,8 @@ export function DepositForm({
         )}
 
       <Button
-        loading={
-          isSubmitting ||
-          !["final", "idle", "confirmed", "error"].includes(status || "")
-        }
+        loadingText="Depositing..."
+        loading={isSubmitting || isAmmUpdating}
```

Comment:
so are we ignoring the tx now 🤔 ?

Reply:
actually the `isAmmUpdating` is `true` when the status is something being processed.